tox.ini

```diff
@@ -37,7 +35,8 @@ commands =
 
 [testenv:quality]
 deps =
-    Django>=1.11,<2.0
+    django111: Django>=1.11,<2.0
+    django20: Django>=2.0,<2.1
```

Comment:
Do we actually run the quality checks with 2 different Django versions?  Doesn't seem like we need two cases for this.